README.md

```diff
@@ -21,12 +21,12 @@ terraform {
 }
 
 module "my_workerpool" {
-  source = "github.com/spacelift-io/terraform-aws-spacelift-workerpool-on-ec2?ref=v2.12.0"
-
-  configuration = <<-EOT
-    export SPACELIFT_TOKEN="${var.worker_pool_config}"
-    export SPACELIFT_POOL_PRIVATE_KEY="${var.worker_pool_private_key}"
-  EOT
+  source = "github.com/spacelift-io/terraform-aws-spacelift-workerpool-on-ec2?ref=v3.0.0"
+  
+  secure_strings = {
```

Comment:
should we somehow signify that this map is being exported as env vars? idk, maybe instead of secure_string, `exported_variables` or `env_vars`, `env_variables` etc.

Reply:
Youre right, I will make this change. Thinking `secure_env_vars` so they are aware they are saved in a safe manner.